Documentation/kubernetes-networking.md

```diff
@@ -0,0 +1,91 @@
+
+Controller Node Inbound
+
+| Protocol | Port Range | Source                       | Purpose                |
+-----------|------------|------------------------------|------------------------|
+| TCP      | 443        | Worker Nodes & API Consumers | Kubernetes API server. |
+
+Worker Node Inbound
+
+| Protocol | Port Range  | Source                         | Purpose                                                                |
+-----------|-------------|--------------------------------|------------------------------------------------------------------------|
+| TCP      | 10250       | Control Nodes                  | Worker node Kubelet healthcheck port.                                  |
+| TCP      | 30000-32767 | ALL or External Load-balancers | Default port range for [external service][external-service] node ports |
+| TCP      | ALL         | Worker & Control Nodes         | Intra-cluster communication (unnecessary if flannel is used)           |
+| UDP      | 8285        | Worker Nodes                   | flannel overlay network (only required if using flannel)               |
+
+etcd Node Inbound
+
+| Protocol | Port Range | Source        | Purpose                                                  |
+-----------|------------|---------------|----------------------------------------------------------|
+| TCP      | 2379       | Control Nodes | etcd server client API                                   |
+| TCP      | 2379       | Worker Nodes  | etcd server client API (only required if using flannel). |
+| TCP      | 2380       | etcd Nodes    | Used for etcd intra-cluster communication.               |
+
+[external-service]: http://kubernetes.io/v1.0/docs/user-guide/services.html#external-services
+
+## Advanced Configuration
+
+The Coreos Kubernetes documentation will describe either using cloud-provider networking integration (e.g. AWS, GCE), or a software-defined overlay network (e.g. [flannel][coreos-flannel]) to manage the Kubernetes Pod Network. However, in some cases a deployer may want to make use of existing network infrastructure to manage the Kubernetes network themselves.
+
+[coreos-flannel]: https://coreos.com/flannel/docs/latest/flannel-config.html
+
+### Container-to-Container Communication
+
+No specific configuration is necessary, as the networking between containers in a pod is managed by Kubernetes itself. This is accomplished by linking all containers in a pod into the same local network.
+
+### Pod-to-Pod Communication
+
+Each pod in the Kubernetes cluster will be assigned an IP that is expected to be routable from all other hosts and pods in the Kubernetes cluster. Generally this will be accomplished by assigning an IP range to each host, and allocating an IP from that range to each launched pod.
```

Comment:
Can we give a quick hint at how this is accomplished? Some probably wrong examples:

"Generally this will be accomplished by **your XXX** assigning an IP range to each host"

"Each pod in the Kubernetes cluster will be assigned an IP **from the XXX** that is expected to be routable"

Is the item a DHCP server? or a network administrator?